src/lib/jsonParse/jsonParse.cpp

```diff
@@ -286,10 +286,9 @@ void eatCompound
       }
 
       containerP->add(orion::ValueTypeString, nodeName, nodeValue);
-      LM_T(LmtCompoundValue, ("Added string '%s' (value: '%s') under '%s'",
+      LM_T(LmtCompoundValue, ("Added string '%s' (value: '%s')",
                               nodeName.c_str(),
-                              nodeValue.c_str(),
-                              containerP->cpath()));
+                              nodeValue.c_str()));
```

Comment:
I'm assuming the 'container' has been removed.
And yes, that was indeed useful during debugging. Probably not needed anymore.

Reply:
Correct. 'container' has been removed.

NTC  (I guess)

Reply:
NTC